druid/src/window.rs

```diff
@@ -53,6 +54,9 @@ pub struct Window<T> {
     pub(crate) handle: WindowHandle,
     pub(crate) timers: HashMap<TimerToken, WidgetId>,
     // delegate?
+    /// A stack of modal widgets. Only the top widget on the stack gets user interaction events.
```

Comment:
It's unclear what *top of the stack* means in terms of a vector. I'd rather use first/last which are well understood.